docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -93,3 +93,29 @@ Workaround: Delete the existing {project} VM and start again.
 $ minishift delete --force
 $ minishift start
 ----
+
+[[Remove-password-from-keychain]]
+== Removing registration password from OS native keychain
+
+=== Windows
+
+In windows the password is stored using *Credential Manager*.
+To delete the stored password run the following command in command prompt:
+
+----
+CMD> cmdkey /delete:minishift:<username>
+----
+
+=== GNU/Linux
```

Comment:
I don't personally mind if we keep "GNU/Linux", but it's worth noting that the OpenShift documentation refers to the operating system as "Linux". This is why I've used "Linux" previously in the documentation -- to keep it consistent.

Reply:
Note: You're also switching between "GNU/Linux" and "Linux" within the same section -- use one or the other, not both, please. :)